test/e2e/resources/identity_provider_config.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: eks.services.k8s.aws/v1alpha1
+kind: IdentityProviderConfig
+metadata:
+  name: $CR_NAME
+spec:
+  clusterName: $CLUSTER_NAME
+  oidc:
+    clientID: client
+    identityProviderConfigName: $IDP_NAME
+    issuerURL: https://some-url.com/
+    usernameClaim: user
+    groupsClaim: group
+  tags:
+    tkey: tval
```

Comment:
Can we also support tag updates? I think we already have most of the logic in `pkg/util`

Reply:
[tagging resource requires arn](https://docs.aws.amazon.com/cli/latest/reference/eks/tag-resource.html). But identity provider config does not have arn associated with it. Even in the UI, there is no way to update its tags. So, I don't think, we can support tag updates.

Reply:
I have made `tags` field immutable now.